data/json/items/comestibles.json

```diff
@@ -2422,12 +2422,12 @@
         "symbol" : "%",
         "quench" : -1,
         "healthy" : 1,
-        "nutrition" : 40,
+        "nutrition" : 24,
```

Comment:
Drop the nerf then. Fix nutrition in a separate PR in a way that doesn't make those recipes useless except for skill grinding.